modules/wordads/class-wordads.php

```diff
@@ -313,52 +344,53 @@ public function resource_hints( $hints, $relation_type ) {
 	 *
 	 * @return [type] [description]
 	 */
-	function insert_head_meta() {
+	public function insert_head_meta() {
 		if ( self::is_amp() ) {
 			return;
 		}
-		$themename = esc_js( get_stylesheet() );
 		$pagetype  = intval( $this->params->get_page_type_ipw() );
 		$data_tags = ( $this->params->cloudflare ) ? ' data-cfasync="false"' : '';
 		$site_id   = $this->params->blog_id;
 		$consent   = intval( isset( $_COOKIE['personalized-ads-consent'] ) );
-		echo <<<HTML
```

Comment:
Is there any particular reason we're replacing the heredocs? Notably the raw HTML isn't echoing the attributes now.

<img width="594" alt="Screen Shot 2020-06-25 at 11 33 00 AM" src="https://user-images.githubusercontent.com/273708/85780511-8abcaa00-b6d9-11ea-9ac8-8389c8fb7118.png">

Reply:
Fixed in d6c424a.

Replaced the heredoc to make it a bit clearer that those variables are being escaped; they weren't all before.